Translate 2016 Fukuoka Ruby Award (zh_cn)

Zane5 opened the pull request:
Related #1194

![screen shot 2015-10-29 at 17 28 55](https://cloud.githubusercontent.com/assets/245165/10815380/9dfd8a2c-7e66-11e5-83ee-55ef4cf85222.png)

JuanitoFatas commented on zh_cn/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md:
```diff
@@ -0,0 +1,55 @@
+lang: zh_cn
+---
+
+亲爱的 Ruby 爱好者,
```
Please use Chinese comma.

JuanitoFatas commented on zh_cn/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md:
```diff
@@ -0,0 +1,55 @@
+
+[http://www.digitalfukuoka.jp/uploads/event_detail/file/210/RubyAward_ApplicationForm_2016.doc](http://www.digitalfukuoka.jp/uploads/event_detail/file/210/RubyAward_ApplicationForm_2016.doc)
+
+请发送申请表格到 award@f-ruby.com
```
Missing period `。`?

JuanitoFatas commented:
Other than my comments, LGTM. :+1: 

P.S. [Commit messages could be improved](https://github.com/ruby/www.ruby-lang.org/wiki#commits-and-commit-messages) :blush:

hlcfan commented:
:boom: Thank you!

stomar commented:
Yep, like `Translate 2015-10-13-fukuoka-ruby-award-2016 (zh_cn)` or e.g. `Translate 2016 Fukuoka Ruby Award (zh_cn)`, not more than 50 characters if possible.

Please amend your commit for these tiny changes (`git commit --amend`) and also fix the commit message.

Zane5 commented:
I have fixed Chinese comma and changed the commit with `git commit --amend`
but I can not push it.
need some help.

```
± % git status
On branch zh_cn/fukuoka-ruby-award-2016
nothing to commit, working directory clean
```

```
git push zane5 zh_cn/fukuoka-ruby-award-2016                                     

To git@github.com:zane5/www.ruby-lang.org.git
 ! [rejected]        zh_cn/fukuoka-ruby-award-2016 -> zh_cn/fukuoka-ruby-award-2016 (non-fast-forward)
error: failed to push some refs to 'git@github.com:zane5/www.ruby-lang.org.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

```
± %  git pull origin master                                                          !2371
Enter passphrase for key '/home/vagrant/.ssh/id_rsa':
From github.com:ruby/www.ruby-lang.org
 * branch            master     -> FETCH_HEAD
Already up-to-date.
```

JuanitoFatas commented:
You need to force push this branch to update commit message: `git push zane5 zh_cn/fukuoka-ruby-award-2016 -f`, [see docs on --force option](https://www.kernel.org/pub/software/scm/git/docs/git-push.html).

Zane5 commented:
@JuanitoFatas thanks :)

stomar commented:
:+1:

JuanitoFatas commented:
:+1:
